Tests/Indicators/ArnaudLegouxMovingAverageTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
```

Comment:
Please add the QC license banner and XML comments, otherwise the build warning count keeps growing :)